catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -272,21 +283,92 @@ define([
                     data = this.model.toJSON();
                 }
                 data.name = this.getNodeName();
+                data.remote = this.isRemote();
```

Comment:
Shouldn't need this field since the data from the model will already contain a `localNode` field which gives you the same information.